mediathread/templates/clientside/collectionwidget.mustache

```diff
@@ -116,6 +109,31 @@
     <div class="collection-assets annotation-embedding">
         <div id="asset_table" class="asset-table"></div>
```

Comment:
The old loader needs to be replaced with the one from the composition space. Right above the `asset_table` here, you can add in:
```
        <div class="ajaxloader">
            <div class="message">
            <img alt="Loading..." src="{{#staticUrl}}{{/staticUrl}}img/ajax-loader.gif">
            <br /><br />
            <div>Refreshing the collection</div>
            </div>
        </div>
```

Reply:
@sdreher when I had the ajaxloader code in the collectionwidget.mustache, it would display the loader but only to about half way down. I placed the loader in the modal dialog div so that the loader styles would apply to the whole modal. I can place it back in here but the styling would have to change a little.